backend/clubs/views.py

```
@@ -2616,39 +2620,33 @@ def create_tickets(self, request, *args, **kwargs):
                 content:
                     application/json:
                         schema:
-                            type: object
-                            properties:
-                                detail:
-                                    type: string
+                            type: array
+                            items:
+                                type: object
+                                additionalProperties:
+                                    type: array
+                                    items:
+                                        type: string
         ---
         """
         event = self.get_object()
+        quantities = request.data.get("quantities", [])
```

should we throw an error if `quantities` is empty?